openfisca_survey_manager/scripts/build_collection.py

```diff
@@ -134,12 +144,19 @@ def build_survey_collection(
             os.mkdir(collections_directory)
         collection_json_path = os.path.join(collections_directory, "{}.json".format(collection_name))
         survey_collection.dump(json_file_path = collection_json_path)
-        surveys = [survey for survey in survey_collection.surveys if survey.name.endswith(str(survey_suffix))]
+        surveys = []
+        for survey in survey_collection.surveys:
+            # TODO: Check with Mahdi why all suveys of the same year was added to the collection and not only the one we provide in parameters ?
```

Comment:
@benjello Do you remember why ?

Reply:
Which parameters are you talking about ?
Aren't they in `data_directory_path_by_survey_suffix` ?

Reply:
I mean, in `def build_collection(` we provide a `collection_name` that is used to build the `survey_name` so why the original loop is rebuiding all the surveys in the collection for the year, as `survey_suffix` contains the year.

Reply:
I don't remember @benoit-cty